portfolio/src/main/java/com/google/sps/data/BlogHashMap.java

```diff
@@ -24,44 +24,47 @@
 import java.util.Set;
 
 public final class BlogHashMap {
-  private static final String[] internalTags = {"#general", "#wellbeing", "#music", "#games"};
+  private final String[] INTERNAL_TAGS = {"#general", "#wellbeing", "#music", "#games"};
+  private Map<String, LinkedList<BlogMessage>> map;
+
+  // Returns whether a message's tag is supported internally.
+  private boolean tagIsSupported(BlogMessage message) {
+    boolean isTagSupported = false;
+    for (int i = 0; i < INTERNAL_TAGS.length; i++){
+      if (message.getTag().equals(INTERNAL_TAGS[i])){
+        isTagSupported = true;
```

Comment:
Consider replacing "isTagSupported = true;" to "return true;" to return early and convey a more clear message to reader: "as long as one tag is matched, it is supported." Also, that way, you don't need to define the "isTagSupported" local variable because you can simply return false when reaching the end of the function.

Reply:
Also, when you start to support more tags (which means the array becomes larger), you could consider to use a HashMap for the INTERNAL_TAGS array to check existence/matching in O(1) vs. O(n). You don't have to do it in this PR, but keep that in mind or add a TODO comment for next change, etc.

Reply:
Got it